pkgs/development/haskell-modules/make-package-set.nix

```diff
@@ -118,10 +118,16 @@ let
       '';
   };
 
-  hackage2nix = name: version: self.haskellSrc2nix {
+  all-cabal-hashes-component = name: version: pkgs.runCommand "all-cabal-hashes-component-${name}-${version}" {} ''
+    tar --wildcards -xzvf ${all-cabal-hashes} \*/${name}/${version}/${name}.{json,cabal}
+    mkdir -p $out
+    mv */${name}/${version}/${name}.{json,cabal} $out
+  '';
+
+  hackage2nix = name: version: let component = all-cabal-hashes-component name version; in self.haskellSrc2nix {
```

Comment:
Did you test whether those changes actually work in the `release-17.09` branch?

Reply:
It looks like the changes only affect macOS users either way, and I don't have a machine running that OS to test on.

Reply:
Otherwise `pkgs/development/haskell-modules/make-package-set.nix` is the only file that references `all-cabal-hashes` and they were updated in parallel.

Reply:
Also this has been backported to `release-17.03` https://github.com/NixOS/nixpkgs/pull/32772

Reply:
I guess this means the answer is *no*, you didn't test it.

Please do.

Reply:
Okay, done.